assets/terraform-modules/bootkube/conditional.tf

```diff
@@ -15,6 +15,14 @@ resource "local_file" "calico" {
     pod_cidr                        = var.pod_cidr
     enable_reporting                = var.enable_reporting
     blocked_metadata_cidrs          = var.blocked_metadata_cidrs
+    failsafe_inbound_host_ports = var.failsafe_inbound_host_ports != null ? [
```

Comment:
I don't understand this change.

Reply:
Previously we were just setting `failsafeInboundHostPorts` in calico-host-protection chart as []. Which should not be case, for more information https://github.com/kinvolk/lokomotive/pull/911#discussion_r484406367